src/utils/menu.js

```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
     sublinks: [
       {
-        name: 'Honey Faucet',
-        link: 'https://faucet.1hive.org/',
-        description: 'Register with BrightId to claim Honey'
+        name: 'Honeypot',
+        link: 'https://1hive.org',
+        description: 'DAO Governance'// - Create and stake on proposals'
       },
       {
-        name: '1hive DAO Interface',
-        link: 'https://1hive.org',
-        description: 'Create Proposals and Stake Honey'
+        name: 'Honey Faucet',
+        link: 'https://faucet.1hive.org/',
+        description: 'Claim Free Honey'// - Register with BrightId to claim Honey'
       },
       {
-        name: 'Honeyswap Interface',
+        name: 'Honeyswap',
         link: 'https://honeyswap.org/',
-        description: 'Swap tokens and supply liquidity'
+        description: 'Decentralized Exchange' //Swap & Supply Liquidity - Swap tokens and supply liquidity. Create new pools'
       },
       {
-        name: 'Honeyswap Analytics',
-        link: 'https://info.honeyswap.org',
-        description: 'Honeyswap analytics and historical data'
+        name: 'Honeycomb',
+        link: 'https://hny.farm/',
+        description: 'Yield Farming'// - Stake Honeyswap LP tokens to claim Honey.'
       }
     ]
   },
   {
-    name: 'Documentation',
+    name: 'External Apps',
     sublinks: [
       {
-        name: '1hive Handbook',
-        link: '/docs/dao',
-        description: 'Learn about 1hive and how to contribute'
+        name: 'BrightID',
+        link: 'https://www.brightid.org/',
+        description: 'Verify for Faucet'
       },
-
       {
-        name: 'Honeyswap Docs',
-        link: '/docs/honeyswap'
+        name: 'MetaMask',
+        link: 'https://metamask.io/',
+        description: 'Crypto wallet'
```

Comment:
I don't think we need to link to Metamask. I think it's safe to assume that anyone who is on this page already has some really basic knowledge and a wallet at hand.

Reply:
I never had a metamask until I came to 1hive

Reply:
i don't think that there is a need to link to metamask in our page, if you think in all the dapps there is no link to metamsk into all of thems, i would suggest to remove it

Reply:
ok, removed.